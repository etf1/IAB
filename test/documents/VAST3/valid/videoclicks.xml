<!-- <Creative> with all differents <VideoClicks> values -->
<VAST version="3.0">
    <Ad id="601364">
        <InLine>
            <AdSystem>Acudeo Compatible</AdSystem>
            <AdTitle>VAST 2.0 Instream Test 1</AdTitle>
            <Description>VAST 2.0 Instream Test 1</Description>
            <Error>http://myErrorURL/error</Error>

            <Impression>http://myTrackingURL/impression</Impression>
            <Creatives>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <ClickThrough>http://www.google.fr</ClickThrough>
                        </VideoClicks>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <ClickThrough id="myId">http://www.google.fr</ClickThrough>
                        </VideoClicks>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <ClickTracking>http://www.google.fr</ClickTracking>
                        </VideoClicks>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <CustomClick>http://www.google.fr</CustomClick>
                        </VideoClicks>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <ClickThrough>http://www.google.fr</ClickThrough>
                            <ClickTracking>http://www.google.fr</ClickTracking>
                        </VideoClicks>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <ClickThrough>http://www.google.fr</ClickThrough>
                            <ClickTracking>http://www.google.fr</ClickTracking>
                            <ClickTracking>http://www.google.fr</ClickTracking>
                        </VideoClicks>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <ClickThrough>http://www.google.fr</ClickThrough>
                            <CustomClick>http://www.google.fr</CustomClick>
                        </VideoClicks>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <ClickThrough>http://www.google.fr</ClickThrough>
                            <CustomClick>http://www.google.fr</CustomClick>
                            <CustomClick>http://www.google.fr</CustomClick>
                        </VideoClicks>
                    </Linear>
                </Creative>
                <Creative>
                    <Linear>
                        <Duration>01:00:00</Duration>
                        <VideoClicks>
                            <ClickThrough>http://www.google.fr</ClickThrough>
                            <CustomClick>http://www.google.fr</CustomClick>
                            <CustomClick id="test1">http://www.google.fr</CustomClick>
                            <ClickTracking>http://www.google.fr</ClickTracking>
                            <ClickTracking id="test2">http://www.google.fr</ClickTracking>
                        </VideoClicks>
                    </Linear>
                </Creative>
            </Creatives>
        </InLine>
    </Ad>
</VAST>
