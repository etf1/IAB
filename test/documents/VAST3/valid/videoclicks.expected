{
  "version": "3.0",
  "ads": [
    {
      "adType": 1,
      "adTitle": "VAST 2.0 Instream Test 1",
      "description": "VAST 2.0 Instream Test 1",
      "adSystem": {
        "name": "Acudeo Compatible"
      },
      "id": "601364",
      "error": "http://myErrorURL/error",
      "impressions": [
        {
          "uri": "http://myTrackingURL/impression"
        }
      ],
      "creatives": [
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "clickThrough": {
              "uri": "http://www.google.fr"
            }
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "clickThrough": {
              "uri": "http://www.google.fr",
              "id": "myId"
            }
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "clickTrackings": [
              {
                "uri": "http://www.google.fr"
              }
            ]
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "customClicks": [
              {
                "uri": "http://www.google.fr"
              }
            ]
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "clickThrough": {
              "uri": "http://www.google.fr"
            },
            "clickTrackings": [
              {
                "uri": "http://www.google.fr"
              }
            ]
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "clickThrough": {
              "uri": "http://www.google.fr"
            },
            "clickTrackings": [
              {
                "uri": "http://www.google.fr"
              },
              {
                "uri": "http://www.google.fr"
              }
            ]
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "clickThrough": {
              "uri": "http://www.google.fr"
            },
            "customClicks": [
              {
                "uri": "http://www.google.fr"
              }
            ]
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "clickThrough": {
              "uri": "http://www.google.fr"
            },
            "customClicks": [
              {
                "uri": "http://www.google.fr"
              },
              {
                "uri": "http://www.google.fr"
              }
            ]
          }
        },
        {
          "creativeType": 1,
          "duration": "01:00:00",
          "videoClicks": {
            "clickThrough": {
              "uri": "http://www.google.fr"
            },
            "clickTrackings": [
              {
                "uri": "http://www.google.fr"
              },
              {
                "uri": "http://www.google.fr",
                "id": "test2"
              }
            ],
            "customClicks": [
              {
                "uri": "http://www.google.fr"
              },
              {
                "uri": "http://www.google.fr",
                "id": "test1"
              }
            ]
          }
        }
      ]
    }
  ]
}
